.travis.yml

```diff
@@ -1,15 +1,30 @@
 language: ruby
 cache: bundler
-sudo: false
 
 rvm:
-  - 2.3.3
-  - 2.4.1
+  - 2.5.8
+  - 2.7.2
+env:
+  matrix:
+    - PG_VERSION=11
+
+before_install:
+  - git config --global user.name "Prodder In Travis-CI"
+  - git config --global user.email "prodder@example.com"
+  # install postgres
+  - sudo apt-get install postgresql-client-$PG_VERSION postgresql-server-dev-$PG_VERSION
+  # setup pg_dump
+  - ls -al /usr/lib/postgresql/
+  - sudo ln -sfn /usr/lib/postgresql/$PG_VERSION/bin/pg_dump /usr/bin/pg_dump
+  # start up the specific version of PG
+  - sudo -E sh -c 'service postgresql stop'
+  - sleep 5s
+  - sudo -E sh -c 'service postgresql start $PG_VERSION'
+  - psql -U postgres -d postgres -c 'select setting from pg_settings where name = $m$server_version$m$;'
```

Comment:
Do we need all this setup?

Reply:
When I tried to build with existing config, it fails - https://travis-ci.org/github/enova/obvious_data/jobs/742080832

Let me try using [travis-ci services](https://docs.travis-ci.com/user/database-setup/#using-a-different-postgresql-version)

Reply:
Yeah, this was our 6 years ago workaround for installing specific PostgreSQL versions before Travis officially supported it as a regular dependency.

Reply:
Makes sense. I'll make this a draft PR and work on it. Travis CI queue is painfully long.

I also tried to move to [github actions:](https://github.com/mokpro/obvious_data/pull/2/checks?check_run_id=1764927199), but I'm blocked. Please take a look and suggest something I can try.

Reply:
Thank you for comment @packrat386. Fixed it in https://github.com/enova/obvious_data/pull/4

Closing current PR